activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -690,6 +690,10 @@ def unique_constraints(table_name)
         #   The constraint name. Defaults to <tt>excl_rails_<identifier></tt>.
         # [<tt>:deferrable</tt>]
         #   Specify whether or not the exclusion constraint should be deferrable. Valid values are +false+ or +:immediate+ or +:deferred+ to specify the default behavior. Defaults to +false+.
+        # [<tt>:using</tt>]
+        #   Specify which index method to use when creating this exclusion constraint (e.g. +:btree+, +:gist+ etc).
+        # [<tt>:where</tt>]
```

Comment:
Oh! I had to execute my exclusion constraint manually because I thought the where clause was not supported! 

Should have looked at the original PR 🤦🏻‍♂️